packages/js/src/helpers/createInterpolateElement.js

```diff
@@ -280,10 +277,7 @@ function proceed( conversionMap ) {
 			// Otherwise we're nested and we have to close out the current
 			// Block and add it as a innerBlock to the parent
 			const stackTop = stack.pop();
-			const text = indoc.substr(
-				stackTop.prevOffset,
-				startOffset - stackTop.prevOffset
-			);
+			const text = indoc.substring( stackTop.prevOffset, startOffset );
```

Comment:
I noticed that in this file, many of end index were changed now that it uses `substring()`. What was the reason of changing the end index? Do you mind adding an explanation?